File: TCDNew-master/src/ui.android/lib/models/dtos/PurchaseTaskFinishReq.dart
class PurchaseTaskFinishReq {
  String purchaseTaskId;
  bool isDecline;
  bool doUpload;

  PurchaseTaskFinishReq({this.purchaseTaskId, this.isDecline, this.doUpload});

  toJson() {
    return {
      'purchaseTaskId': purchaseTaskId,
      'isDecline': isDecline,
      'doUpload': doUpload,
    };
  }
}

File: TCDNew-master/src/api/Cen.Wms.Domain.Purchase.Api/Dtos/PurchaseTaskFinishReq.cs
using System;

namespace Cen.Wms.Domain.Purchase.Api.Dtos
{
    public class PurchaseTaskFinishReq
    {
        public Guid PurchaseTaskId { get; set; }
        public bool IsDecline { get; set; }
        public bool DoUpload { get; set; }
    }
}